analysis/source/multivar_draw/SConscript

```diff
@@ -0,0 +1,11 @@
+Import('env')
+import gslab_scons as gs
+
+build  = env['CONFIG']['global']['build']['multivar_draw']
+source = env['CONFIG']['global']['source']['multivar_draw']
+
+env.BuildR(target  = ['#%s/multivar_draw.csv'      % build],
+            source  = ['#%s/multivar_draw.r' % source],
+            log_ext =  'multivar_draw')
+
+env.Install('#release', '#%s/' % build)
```

Comment:
No need to send the .csv to release. Normally, we don't need to commit derived data files, only figures and tables.